index.html

```diff
@@ -323,155 +327,186 @@ <h1>Multi-Post Stories Gain+Glory</h1>
 								<li>
 									<a href="#">
-								<button id="ruby">Ruby on rails</button>
-								</a>
+										<button id="ruby">Ruby on rails</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">RSpec</button>
-								</a>
+										<button id="ruby">RSpec</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Capybara</button>
-								</a>
+										<button id="ruby">Capybara</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Selenium</button>
-								</a>
+										<button id="ruby">Selenium</button>
+									</a>
 								</li>
 							</ul>
 
 						</div>
-			</div>
+					</div>
 
-			<!-- about-me-card  3 -->
-			<div class="about-me-cards">
-				<div class="icon-languages">
-					<img src="images/icon-skills.svg" alt="languages">
-				</div>
-				<div>
-					<h1>
-					   Skills 
-					</h1>
-				</div>
+					<!-- about-me-card  3 -->
+					<div class="about-me-cards">
+						<div class="icon-languages">
+							<img src="images/icon-skills.svg" alt="languages">
+						</div>
+						<div>
+							<h1>
+								Skills 
+							</h1>
+						</div>
 						<div class="abt-frameworks">
 							<ul class="abt-languages">
 								<li>
 									<a href="#">
-								<button id="ruby">Codekit</button>
-								</a>
+										<button id="ruby">Codekit</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">GitHub</button>
-								</a>
+										<button id="ruby">GitHub</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Codepen</button>
-								</a>
+										<button id="ruby">Codepen</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">GitLab</button>
-								</a>
+										<button id="ruby">GitLab</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Terminal</button>
-								</a>
+										<button id="ruby">Terminal</button>
+									</a>
 								</li>
 							</ul>
 
 						</div>
+					</div>
+
+				</div>
 			</div>
+		</section>
+		<section>
+			<div class="form-section">
+				<div> 
+					<h1 class="frm-txt">I'm always interested in hearing about new projects,
+					so if you'd like to chat please get in touch.</h1>
+				</div>
+				<form action="https://formspree.io/f/xwkzwlne" method="POST" >
+					<input class="form-input" type="text" name="name" placeholder="Full Name" maxlength="30" required>
+					<input class="form-input" type="email" name="email" placeholder="Email Address" required>
+					<textarea class="form-input form-input2" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+					<button  type="submit">Get in touch</button>
+				</form>
+				<div class="form-img">
+					<img src="images/Frame.svg" alt="Frame"/>
+					
+				</div>
```

Comment:
- For aesthetic reasons, I suggest that all images that are not part of your page's main content should be added using the `background` property in CSS. Please fix this, you should also consider fixing it for all other images as they may apply 🙏

Reply:
Hello @mckent05 Thank you for taking the time to review my work. 

I will be implementing all the other requested changes. 

However, I would like more clarification on the "for aesthetic reasons" change. [Refer Here for the change requested](https://github.com/jaredatandi/Portfolio/pull/9#discussion_r1010860672). It would be helpful if you provided any and every possible reference as I would like to be more knowledgeable and prevent such issues in the future.
 
Thank you.

Reply:
> Hello @mckent05 Thank you for taking the time to review my work.
> 
> I will be implementing all the other requested changes.
> 
> However, I would like more clarification on the "for aesthetic reasons" change. [Refer Here for the change requested](https://github.com/jaredatandi/Portfolio/pull/9#discussion_r1010860672). It would be helpful if you provided any and every possible reference as I would like to be more knowledgeable and prevent such issues in the future.
> 
> Thank you.

[OPTIONAL] What the previous reviewer was talking about is that whenever an image is to design(just design not passing any message), it is always used from the CSS but if the image carries a message, then it can be used directly from the HTML. kindly consider using the image from the background.

Reply:
Thank you @Olamarx